tests/vm_test.go

```diff
@@ -1581,9 +1582,9 @@ status:
 			_, _, err := clientcmd.RunCommand(testsuite.GetTestNamespace(nil), k8sClient, "create", "-f", vmJson)
 			Expect(err).ToNot(HaveOccurred())
 
-			By("Invoking virtctl start")
-			startCommand := clientcmd.NewRepeatableVirtctlCommand(virtctl.COMMAND_START, "--namespace", vm.Namespace, vm.Name)
-			Expect(startCommand()).To(Succeed())
+			By("Starting the VM")
```

Comment:
`should create and delete via command line` (see the `It`) is now miss leading

Reply:
This test is still using kubectl, so it is partially command line / partially client. Updated it to drop `via command line`.